src/routes/v2/authenticated/review.ts

```diff
@@ -691,6 +709,11 @@ export class ReviewsRouter {
       })
     }
 
+    // Check state here to ensure it's open
+    if (possibleReviewRequest.reviewStatus !== ReviewRequestStatus.Open) {
+      // return 400
```

Comment:
unintended comment?

Reply:
ohyea good catch - this are part of VAPT fixes and shouldn't be here oops